src/bundle/Resources/views/themes/admin/ui/form_fields.html.twig

```diff
@@ -247,10 +255,77 @@
 
 {% block form_errors -%}
     {%- if errors|length > 0 -%}
+        {%- set form_errors_attr = form_errors_attr|default({}) -%}
+        {%- set form_errors_attr = form_errors_attr|merge({class: (form_errors_attr.class|default('') ~ ' ibexa-form-error')|trim}) -%}
         {%- for error in errors -%}
-            <em class="ez-block-config__error">{{ error.message }}</em>
+            <em {% with { attr: form_errors_attr } %}{{ block('attributes') }}{% endwith %}>
+                <svg class="ibexa-icon ibexa-icon--small ibexa-form-error__icon">
+                    <use xlink:href="{{ ibexa_icon_path('warning-triangle') }}"></use>
+                </svg> 
+                {{ error.message }}
+            </em>
         {%- endfor -%}
     {%- endif -%}
 {%- endblock form_errors %}
 
 {% block form_label_errors %}{% endblock form_label_errors %}
+
+{%- block form_widget_simple -%}
+    {%- set is_text_input = type is not defined or type is empty or type == 'text' -%}
+    {%- if is_text_input -%}
+        {%- set attr = attr|merge({class: (attr.class|default('') ~ ' ibexa-input ibexa-input--text')|trim}) -%}
+        {%- set empty_placeholder_for_hiding_clear_btn_with_css = ' ' -%}
+        {%- set attr = attr|merge({placeholder: (attr.placeholder is defined) ? attr.placeholder : empty_placeholder_for_hiding_clear_btn_with_css}) -%}
+        <div class="ibexa-input-text-wrapper">
+    {%- endif -%}
+    {{ parent() }}
+    {%- if is_text_input -%}
+            <button type="button" class="btn ibexa-input__clear ibexa-input-text-wrapper__clear-btn" tabindex="-1">
```

Comment:
bad indent

Reply:
It was indented to match indentation from: 
`<div class="ibexa-input-text-wrapper">` which is a few lines above.
In my opinion, indentation is OK here.